src/course/syllabus/apprenticeship/database/project.md

```diff
@@ -1,5 +1,19 @@
 Your project this week is to build a web app that stores data in a PostgreSQL database.
 
+## Spike
+
+Before you start writing features you need to design the schema for your data. Think about what different things your app needs to store, how they relate to each other, and how you can avoid duplicating information. Record your schema in your `README.md` using Markdown tables.
+
+### Questions to consider
+
+1. What kinds of data relationships are there?
+1. What's a foreign key? How can they help us design schemas with relational data?
+
+### Useful resources
```

Comment:
We might like to include a tool for creating schemas (e.g. https://dbdiagram.io/home). Although, this may be less relevant if we're encouraging them to include their schema in their README.

Reply:
Yeah I did consider that, wanted to keep it simple. I guess those diagrams help people visualise relations—no reason they can't embed an image in the README